.gitlab-ci.yml

```diff
@@ -176,6 +177,32 @@ agent_rpm-x64:
     paths:
       - $OMNIBUS_PACKAGE_DIR
 
+# build Agent package for rpm-x64
+agent_suse-x64:
+  stage: package_build
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/datadog-agent-builders/suse_x64:latest
+  tags: [ "runner:main", "size:large" ]
+  script:
+    # Artifacts and cache must live within project directory but we run omnibus
+    # from the GOPATH (see above). We then call `invoke` passing --base-dir,
+    # pointing to a gitlab-friendly location.
+    - set +x
+    - RPM_GPG_KEY=$(aws ssm get-parameter --region us-east-1 --name ci.datadog-agent.rpm_signing_private_key --with-decryption --query "Parameter.Value" --out text)
+    - printf -- "$RPM_GPG_KEY" | gpg --import --batch
+    - export RPM_SIGNING_PASSPHRASE=$(aws ssm get-parameter --region us-east-1 --name ci.datadog-agent.rpm_signing_key_passphrase --with-decryption --query "Parameter.Value" --out text)
+    - set -x
+    - inv -e agent.omnibus-build --base-dir $OMNIBUS_BASE_DIR/suse
+    - rpm -i $OMNIBUS_PACKAGE_DIR_SUSE/*.rpm
```

Comment:
`OMNIBUS_BASE_DIR` is cached, not sure if this would do more than what's expected, maybe we should point `OMNIBUS_PACKAGE_DIR` and `OMNIBUS_PACKAGE_DIR_SUSE` to disposable paths. Out of the scope of this PR though.